backends/avx/ceed-avx-blocked.c

```diff
@@ -1,18 +1,9 @@
-// Copyright (c) 2017-2018, Lawrence Livermore National Security, LLC.
-// Produced at the Lawrence Livermore National Laboratory. LLNL-CODE-734707.
-// All Rights reserved. See files LICENSE and NOTICE for details.
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level COPYRIGHT and NOTICE files for details.
```

Comment:
```suggestion
// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
```
The copyright date range in `LICENSE` should be updated.